sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -613,6 +613,46 @@ object functions {
   def min(columnName: String): Column = min(Column(columnName))
 
   /**
+   * Aggregate function: returns the exact percentile(s) of the expression in a group at pc with
```

Comment:
let's not add the scala api for this for now. i'm not sure if we want to encourage users to use it -- it is super expensive.